lib/Serialization/Deserialization.cpp

```diff
@@ -2646,7 +2646,8 @@ class swift::DeclDeserializer {
     if (auto *overridden = ctor->getOverriddenDecl()) {
       if (!attributeChainContains<RequiredAttr>(DAttrs) ||
           !overridden->isRequired()) {
-        AddAttribute(new (ctx) OverrideAttr(SourceLoc()));
+        if (!overridden->isConvenienceInit())
```

Comment:
Can you leave a comment behind saying that this doesn't really make sense? Either the convenience init is an override or it isn't. cc @DougGregor

Reply:
Yeah sure, will update shortly.